renderer/components/UI/Dropdown.js

```diff
@@ -118,13 +126,21 @@ export const MenuItem = withTheme(
       </Flex>
     </MenuItemText>
   )
-)
+}
+
+MenuItem.propTypes = {
+  hasChildren: PropTypes.bool.isRequired,
+  hasParent: PropTypes.bool.isRequired,
```

Comment:
Boolean props generally don't need to be `isRequired` since not supplying is the same as defaulting to `false`

Reply:
Done.